src/spec-node/devContainersSpecCLI.ts

```diff
@@ -117,6 +117,9 @@ function provisionOptions(y: Argv) {
 		'skip-feature-auto-mapping': { type: 'boolean', default: false, hidden: true, description: 'Temporary option for testing.' },
 		'skip-post-attach': { type: 'boolean', default: false, description: 'Do not run postAttachCommand.' },
 		'experimental-image-metadata': { type: 'boolean', default: experimentalImageMetadataDefault, hidden: true, description: 'Temporary option for testing.' },
+		'dotfiles-repository': { type: 'string', description: 'Git URL to clone a dotfiles repository from.' },
```

Comment:
Suggestion: Should we use the wordings from https://code.visualstudio.com/docs/devcontainers/containers#_personalizing-with-dotfile-repositories ? 

```suggestion
		'dotfiles-repository': { type: 'string', description: 'URL of a dotfiles Git repository (e.g., https://github.com/owner/repository.git) or owner/repository of a GitHub repository. },
```